README.md

```diff
@@ -11,10 +11,17 @@ web server.
 Running
 =======
 
-First you must build the project.  Note currently the build process has only
-been tested/maintained on Mac/Linux systems.  Assuming you have a compiler,
-all the normal Unix command-line utilities, and have installed Node and npm,
-you can run:
+**One time only**: We recommend you create a new profile for Firefox.  You can do this by finding the Firefox executable and running it with the `-ProfileManager` option, or `-Profile ./loop-dev/` (to create a profile in `./loop-dev/`).  On a Mac you can do this like:
+
+```shell
+$ open /Applications/FirefoxNightly.app --args -Profile ./loop-dev/
+```
+
+When Firefox opens you must go to the URL `about:config` and
+look for the setting `xpinstall.signatures.required`.  This must be set to
```

Comment:
As of today, you also have to set xpinstall.signatures.required to false, and even that still leaves the thing disabled by default.